targets/arduino-zero.json

```
@@ -0,0 +1,6 @@
+{
+    "inherits": ["atsamd21g18a"],
+    "build-tags": ["sam", "atsamd21g18a", "arduino_zero"],
```

You can leave out the build tags defined in atsamd21g18a.json:
```suggestion
    "build-tags": ["arduino_zero"],
```